lib/Raven/Client.php

```diff
@@ -881,9 +1016,12 @@ public function capture($data, $stack = null, $vars = null)
             }
 
             if (!isset($data['stacktrace']) && !isset($data['exception'])) {
-                $data['stacktrace'] = array(
-                    'frames' => Stacktrace::fromBacktrace($this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__, isset($stack['line']) ? $stack['line'] : __LINE__)->getFrames(),
-                );
+                $data['stacktrace'] = [
+                    'frames' => Stacktrace::fromBacktrace(
+                        $this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__,
+                        isset($stack['line']) ? $stack['line'] : __LINE__
```

Comment:
As `__LINE__` macro is used here to set in the stacktrace the line that generated the first frame, wouldn't be better to revert back these lines to just 1? This way the line will be the correct one (the one where the method is called)